documentation/Functions.md

```diff
@@ -394,11 +394,17 @@ The tutorials repository can be found https://github.com/devonfw-tutorials/tutor
 ***
 
 ### adaptTemplatesCobiGen <a name="adaptTemplatesCobiGen"></a>
+This function is used when you want to adapt CobiGen templates.
```

Comment:
This function is used when you want to adapt CobiGen templates. See [CobiGen CLI documentation](https://devonfw.com/website/pages/docs/howto_Cobigen-CLI-generation.asciidoc.html).